### PR TITLE
big updates

### PR DESCRIPTION
**Pull request by jessyrod68**
- updated opening paragraph 
- removed all the build numbers from headers
- deleted outdated content
- added image to line 426 section

### CODE REVIEW

**Comment by jessyrod68** [comment]
[WIP Docs - edits jr.xlsx](https://github.com/MicrosoftDocs/windows-insider/files/3307788/WIP.Docs.-.edits.jr.xlsx)
 We would also like to discuss how to best rearrange the order of the post so they live in catagoires like in the spreadsheet attached.

**Comment by jessyrod68** [comment]
@ItzLevvie hello, 

We want to group the headers into categories to help with the confusion people may have.  We also noticed that this page does not have build numbers and they seem to be doing well. 

https://docs.microsoft.com/en-us/windows-insider/at-home/whats-new-wip-at-home-1809

We would also settle for the build number still included but as a subheader, not in the main header. Let me know if what can be done please. 

Thanks! :)

**Comment by mypil** [comment]
@lizap - Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Comment by ItzLevvie** [comment]
@jessyrod68 Ah then it's fine. I was just mentioning it as internal also adds its own build numbers over at the [20H1 page](https://github.com/MicrosoftDocs/windows-insider/blob/master/wip/at-home/Whats-new-wip-at-home-20h1.md)

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: wip/at-home/Whats-new-wip-at-home-19h1.md [hunk 1]
```diff
@@ -11,21 +11,14 @@ ms.topic: article
 ms.prod: w10
 ---
 
-# What's new for Windows 10 Insider Preview Builds (19H1)
-The [Windows Insider Program](https://insider.windows.com/en-us/) lets you preview builds of the upcoming release of Windows 10. This topic lists all new Windows 10 features for you to try. 
+# What's new for Windows 10 Insider Preview Builds 1903
+The [Windows Insider Program](https://insider.windows.com/en-us/) lets you preview builds of the upcoming release of Windows 10. This topic lists all all of the updates that were part of the 19H1 development now known as the version "1903".
```
Comment (nenonix) [inline wip/at-home/Whats-new-wip-at-home-19h1.md]:
```suggestion
The [Windows Insider Program](https://insider.windows.com) lets you preview builds of the upcoming release of Windows 10. This topic lists all of the updates that were part of the 19H1 development, now known as version "1903".
```

**Pull request merged**